packages/connect/src/filter.ts

```diff
@@ -118,108 +128,106 @@ export class Filter {
       return false
     }
 
-    // Resolve p2p addresses first
```

Comment:
Can we please split the `filterDial` function into smaller ones to increase readability?

Reply:
changed